managed-ledger/src/main/java/org/apache/bookkeeper/mledger/ManagedLedger.java

```diff
@@ -515,9 +515,4 @@ void asyncSetProperties(Map<String, String> properties, final AsyncCallbacks.Upd
      * @param promise
      */
     void trimConsumedLedgersInBackground(CompletableFuture<?> promise);
-
-    /**
-     * Roll current ledger if it is full
-     */
-    void rollCurrentLedgerIfFull();
```

Comment:
@hangc0276 we shouldn't remove a method from an interface directly. we should keep the method and mark it as deprecated.

Reply:
Ok, i move it back.